src/app/reporting/Engine.cpp

```diff
@@ -176,6 +196,10 @@ DataModel::ActionReturnStatus RetrieveClusterData(DataModel::Provider * dataMode
         // the are routed through metadata.
         status = ReadGlobalAttributeFromMetadata(dataModel, readRequest.path, attributeValueEncoder);
     }
+    else if (auto existence_status = ValidateAttributeIsReadable(dataModel, path); existence_status.has_value())
```

Comment:
Followup: the branch above doing IsSupportedGlobalAttributeNotInMetadata does not match the documentation for the data model provider API which claims that it _does_ handle global attributes.  We should reconcile these things.